packages/app-builder-lib/src/packager.ts

```diff
@@ -477,7 +471,7 @@ export class Packager {
     }
   }
 
-  private async installAppDependencies(platform: Platform, arch: Arch): Promise<any> {
+  public async installAppDependencies(platform: Platform, arch: Arch): Promise<any> {
```

Comment:
Minor: Since `installAppDependencies` is isolated to `doPack` now, can we move this to `platformPackager` and use `protected` instead of `public`?

Reply:
`installAppDependencies` has a bunch of dependencies on `Packager` and not sure it should be moved.  If at all, this probably should be a separate refactoring.